tests/rptest/tests/services_self_test.py

```diff
@@ -67,12 +67,24 @@ def test_kgo_repeater(self):
                       segment_bytes=1024 * 1024))
         with repeater_traffic(context=self.test_context,
                               redpanda=self.redpanda,
+                              num_nodes=2,
                               topic=topic,
                               msg_size=4096,
                               workers=1) as repeater:
             repeater.await_group_ready()
             repeater.await_progress(1024, timeout_sec=75)
 
+        # Assert clean service stop.
+        for service in self.test_context.services:
+            for node in service.nodes:
+                cmd = """ps ax | grep -i kgo-repeater | grep -v grep | awk '{print $1}'"""
```

Comment:
Would this be simpler with `pgrep kgo-repeater`? Not sure if that is available everywhere though.